kibana.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "opendistroAlerting",
+  "version": "1.11.0.2",
```

Comment:
Should we bump to 1.12.0.0 ?

Reply:
I will do the version upgrade in another PR. Keeping the version number as 1.11 makes me possible to test the functionality in an ODFE 1.11 cluster after the migration. 😁